website/pages/home/index.jsx

```diff
@@ -1,350 +1,294 @@
+        links={[
+          {
+            text: 'Download',
+            url: '/downloads',
+            type: 'download'
+          },
+          {
+            text: 'Get Started',
+            url: 'https://learn.hashicorp.com/nomad',
+            type: 'outbound'
+          }
+        ]}
+      />
 
-          <TwoColumnList
-            items={[
-              {
-                title: '1. Declare Jobs',
-                details:
-                  '<p>Users compose and submit high-level job files. Nomad handles the scheduling and upgrading of the applications over time.</p><p>This flexibility makes it easy to deploy one container, dozens of containers, or even <a href="https://www.hashicorp.com/c1m/">millions</a>.</p>'
-              },
-              {
-                title: '2. Plan Changes',
-                details:
-                  'With built-in dry-run execution, Nomad shows what scheduling decisions it will take before it takes them. Operators can approve or deny these changes to create a safe and reproducible workflow.'
-              },
-              {
-                title: '3. Run Applications',
-                details:
-                  'Nomad runs applications and ensures they keep running in failure scenarios. In addition to long-running services, Nomad can schedule batch jobs, distributed cron jobs, and parameterized jobs.'
-              },
-              {
-                title: '4. Monitor Progress',
-                details:
-                  'Stream logs, send signals, and interact with the file system of scheduled applications. These operator-friendly commands bring the familiar debugging tools to a scheduled world.'
-              }
-            ]}
-          />
-        </div>
-      </section>
+      <FeaturesList
+        title="Why Nomad?"
+        items={[
+          {
+            title: 'Simple and Lightweight',
+            content:
+              'Single 35MB binary that integrates into existing infrastructure.  Easy to operate on-prem or in the cloud with minimal overhead.',
+            icon: require('./img/why-nomad/simple-and-lightweight.svg')
+          },
+          {
+            title: 'Flexible Workload Support',
+            content:
+              'Orchestrate applications of any type - not just containers. First class support for Docker, Windows, Java, VMs, and more.',
+            icon: require('./img/why-nomad/flexible-workload-support.svg')
+          },
+          {
+            title: 'Modernize Legacy Applications without Rewrite',
+            content:
+              'Bring orchestration benefits to existing services. Achieve zero downtime deployments, improved resilience, higher resource utilization, and more without containerization.',
+            icon: require('./img/why-nomad/modernize-legacy-applications.svg')
+          },
+          {
+            title: 'Easy Federation at scale',
```

Comment:
```suggestion
            title: 'Easy Federation at Scale',
```

Minor capitalization nit.